recipes/breakfast/arabic_omelette.md

```diff
@@ -0,0 +1,35 @@
+# Arabic Omelette
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-25%25-green.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-10min-brightgreen.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Very%20Positive-brightgreen.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Very%20Cheap-brightgreen.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-False-red.svg)
+
+ **First cooked by:**
+[@Karyum](https://github.com/karyum)
+
+ **Date:** Week 5 FACN2
+ **Author of Recipe:**
+#MrsSaliba
+
+### Ingredients
+- 2 small eggs
+- a handful of parsley
+- a handful of green/spring onions
+- 1 teaspoon zaatar ( or to taste )
+- salt and pepper
```

Comment:
I've heard that arabic omelettes contain mint. Thoughts?

Reply:
it's like different accents mate same language but some words change.
and i do say get creative